ui/cypress/support/commands.ts

```diff
@@ -225,6 +227,75 @@ export const deleteOrg = (id: string) => {
   return cy.request('DELETE', `/chronograf/v1/organizations/${id}`)
 }
 
+export const createInfluxDBUser = (
```

Comment:
FYI: It would be great if you could also disambiguate the existing `createUser` and `deleteUser` commands, they could be renamed to `createChronografUser` and  `deleteChronografUser` respectively.

Reply:
Functions renamed.